benchmark/README.md

```diff
@@ -0,0 +1,47 @@
+# K6 loadtesting on Microcks mock services
+
+1. Start you Microck instance 
+2. Import OpenBanking.org.uk APIs / Read/Write API - Account and Transaction.
+3. Import MicrocksIO Samples APIs / petstore-api-postman v. 1.0
+
+## env vars
+```
+export MICROCKS_BASE_URL=http://172.31.243.54:8080
+export PROMETHEUS_RW_URL=http://172.31.243.54:9080/api/v1/write
+export K6_VERSION=0.48.0
```

Comment:
Was wondering what was this IP address? Is it the one of a VM or Kube node running Microcks?

Reply:
Hi it's just the ip address of my wsl instance, since i'm running k6 in a container it needs to point to the microcks spring boot instance running on localhost from inside the k6 docker container.

Reply:
yes microcks base url is the ip/url of microcks and the prometheus url is the one of prometheus.